.github/workflows/get_modular_resources.py

```diff
@@ -52,10 +53,18 @@ async def main():
                     "api": model.api,
                     "last_updated": time_now
                 }
-                insert_resp = typesense_client.collections['modular_resources'].documents.upsert(
+                insert_resp = typesense_client.collections['resources'].documents.upsert(
         json_m)
                 print(insert_resp)
 
+    # Get builtin resources from typesense.json
```

Comment:
```suggestion
    # Get built-in resources from typesense.json
```
[nit] Per Webster this should be hyphenated. I suppose we can't change it in the code bits because that's an RDK-side thing?

Reply:
Which code bits?

Reply:
If you search the diff for "builtin" three things come up, relating to module IDs.

Reply:
I think I could change it but I think this is in line to how it appears in shortened links? so probably fine